src/hoa-renderer.js

```diff
@@ -182,9 +182,9 @@ HOARenderer.prototype.setRotationMatrixFromCamera = function(camera) {
   if (!this._isRendererReady)
     return;
 
-  // Use the camera object's local transform matrix for the rotation.
+  // Use the camera object's world transform matrix for the rotation.
   // See: https://threejs.org/docs/#api/core/Object3D
-  this._hoaRotator.setRotationMatrix4(camera.matrix.elements);
+  this._hoaRotator.setRotationMatrix4(camera.matrixWorld.elements);
```

Comment:
For HOARenderer, the change is fine, but we need to be careful about the FOA because people have been using the interface for a while. So, this will be a breaking change around V1.

Are we completely sure about this rotation? If so, I guess we need a test for it.